src/components/views/rooms/RoomPreviewBar.tsx

```diff
@@ -76,6 +79,8 @@ interface IProps {
 
     canPreview?: boolean;
     previewLoading?: boolean;
+
+    // The room to be previewed
```

Comment:
Can you expand this please. As a user of this component, I would like to know when it should be specified, and what the effects of not specifying it are.

See the documentation on `roomAlias` for example: it gives a bit of information on how it will be used.

Reply:
comment is updated to mention that room passed to `RoomPreviewBar` is peekable, maybe that is okay